carousel/core/calculators.py

```diff
@@ -161,7 +162,8 @@ def get_covariance(self, datargs, outargs, vargs, datvar, outvar):
         LOGGER.debug('covariance:\n%r', cov)
         return cov
 
-    def calculate(self, calc, formula_reg, data_reg, out_reg,
+    @classmethod
```

Comment:
[`calculate()`](https://github.com/SunPower/Carousel/blob/953a7d8725e30f39e8ed3ed002a1c657eac4d6f9/carousel/core/calculators.py#L165-L166) is a class method because it doesn't ever use `self` but it does use `cls` when it uses the static method [`get_covariance()`](https://github.com/SunPower/Carousel/blob/953a7d8725e30f39e8ed3ed002a1c657eac4d6f9/carousel/core/calculators.py#L105-L106). PyCharm told me to do it.